backend/app/services/realtime_session_service.py

```diff
@@ -185,6 +206,13 @@ async def get_realtime_session(self, session: Session, user_profile: UserProfile
         language = conversation_scenario.language_code.value
 
         logging.info(f'Final prompt:\n{instructions}')
+        # Write the instructions to a text file for debugging/auditing
+        instructions_path = os.path.join(
+            os.path.dirname(__file__), '..', 'logs', f'instructions_{session.id}.txt'
+        )
+        os.makedirs(os.path.dirname(instructions_path), exist_ok=True)
+        with open(instructions_path, 'w', encoding='utf-8') as f:
+            f.write(instructions)
```

Comment:
I meant to only use this code in local development. Otherwise the server will fill up with text files over time. Let's either remove this or add a Boolean environment variable that turns it off by default.